components/list/list-item-generic-layout.js

```diff
@@ -49,12 +67,12 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 				grid-row: 1 / 2;
 			}
 			::slotted([slot="outside-control"]) {
-				width: 42px;
+				width: 2.1rem;
```

Comment:
Fixed in https://github.com/BrightspaceUI/core/pull/678 (as you know since you wrote it.)